Automattic/wp-calypso
# Google My Business: Migrate styles to webpack

Fixes #27534 

#### Changes proposed in this Pull Request

* This PR removes the individual stylesheet imports from the main stylesheet and imports them via webpack within the individual components/blocks.
* Doing so loads the relevant CSS only when necessary and reduces the size of the main Calypso stylesheet.

#### Testing instructions

* Switch to this PR on a Business site and navigate to `/google-my-business/[site-url]`
* Do a visual check to ensure nothing has changed or gone missing
* Connect to Google My Business and do another visual check for out of place elements/styles
